app/lib/back-helpers.js

```diff
@@ -86,10 +86,15 @@ const addBackNavigationForErrorCondition = (request) => {
   }
 }
 
+const getBackLinkToSamePage = (request) => {
+  return request.headers?.referer ?? ''
+}
```

Comment:
Does an href (backlink) of '' just redisplay the same page again, or does it do anything weird?

Reply:
it adds a #, so doesn't work